tests/channel_testing/helpers.py

```diff
@@ -44,7 +61,15 @@ class Starter(ProcessStarter):
         if user and password:
             env["TESTPWD"] = f"{user}:{password}"
 
-        def startup_check(self):
+        def startup_check(self):  # type: ignore
```

Comment:
type checker doesn't like the way we override pattern, startup_check but that's an upstream thing.